src/WiFiSpiClient.h

```diff
@@ -65,6 +65,12 @@ class WiFiSpiClient : public Client {
 
     using Print::write;
 
+    // Return the IP address of the host who sent the current incoming packet
+    virtual IPAddress remoteIP();
```

Comment:
Hi, I am not sure the functions should be _virtual_. Did you declare them so on purpose?

Reply:
Honestly, I don't know, I just copied them from the WiFiSpiUdp library and they work, but I'm not well versed with writing libraries, so "virtual ł could be unnecessary or even wrong.

Reply:
Ok, please remove the virtual keyword and commit it. I am not a C++ expert but I think there is no reason for this. Thanks